### PR TITLE
metrics: add solana to tx metrics

### PR DESCRIPTION
**Pull request by jeff-dude**
broke out into separate CTEs in case we add more in the future

### CODE REVIEW

File: dbt_subprojects/daily_spellbook/models/_metrics/transactions/metrics_transactions_daily.sql [hunk 1]
```diff
@@ -9,39 +9,58 @@
...
+), solana as (
+    select
+        'solana' as blockchain
+        , block_date
+        , id as tx_hash
+    from
+        {{ source('solana', 'transactions') }}
+    where
+        1 = 1
+        and block_date is not null --200m+ rows of nulls
```
Comment (jeff-dude) [inline dbt_subprojects/daily_spellbook/models/_metrics/transactions/metrics_transactions_daily.sql:45]: this is an odd finding here
either of you know why there are so many rows with null `block_date`?

**Comment by jeff-dude** [comment]
(need to remove short timeframe prior to merge)

File: dbt_subprojects/daily_spellbook/models/_metrics/transactions/metrics_transactions_daily.sql (hunk 1 above)
Comment (0xRobin) [reply dbt_subprojects/daily_spellbook/models/_metrics/transactions/metrics_transactions_daily.sql:80]: no clue, sadly :confused:

Comment (0xBoxer) [reply dbt_subprojects/daily_spellbook/models/_metrics/transactions/metrics_transactions_daily.sql:80]: solscan seems equally confused: https://solscan.io/block/31748087 

probably some client issue, looks like that block slot range that doesn't have dates attached to it was 4 years ago, a few of them have timestamps and are dated around that time: https://solscan.io/block/38877915 

Can safely ignore from my POV

Comment (jeff-dude) [reply dbt_subprojects/daily_spellbook/models/_metrics/transactions/metrics_transactions_daily.sql:80]: fwiw, `solana.blocks` also has the nulls (as expected on your findings here). i will leave the filter then, thanks both

**Pull request merged**